spec/services/github_spec.rb

```diff
@@ -2,39 +2,36 @@
 
 describe Github do
   context "when given a repo" do
+    let(:args) { {org_repo: "CodeMontageHQ/codemontage",
+                 repo: "codemontage",
+                 day_begin: Time.new(2014, 10, 01),
+                 day_end: Time.new(2014, 10, 31)} }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 8, 51 should be on its own line.<br>Space inside } missing.